docs/arch.plantuml

```diff
@@ -0,0 +1,38 @@
+@startuml
+center header <b>Ceres Rollup Service Architecture
+queue "unified.metrics.json topic" as input
+card Metrics
+note right: com.rackspacecloud.metrics.rollup.domain.Metric
```

Comment:
Wow, I wish the standard ExternalMetric had been used here and then wrapper types for the accumulation logic.